src/workflows/airqo_etl_utils/bigquery_api.py

```
@@ -309,6 +309,7 @@
         unique_cols = ["id", "network"]
 
         dataframe.reset_index(drop=True, inplace=True)
+        up_to_date_data["last_updated"] = datetime.now(timezone.utc)
```

Ruff (0.8.2)

Undefined name `up_to_date_data`

(F821)